bin/results-generator.sh

```diff
@@ -0,0 +1,138 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```
For those of us using MacOS, this is necessary to use a modern bash.